sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_attribution_v1/schema.yaml

```diff
@@ -76,3 +76,6 @@ fields:
 - mode: NULLABLE
   name: attributed
   type: BOOLEAN
+- mode: NULLABLE
```

Comment:
should a description be added?

Reply:
Definitely!
I've added the description from the Use Story. WE can update it with the link to the 'source of truth' once we have it.